src/redismodule.h

```diff
@@ -672,6 +677,15 @@ typedef struct RedisModuleModuleChange {
     int32_t module_version; /* Module version. */
 } RedisModuleModuleChangeV1;
 
+#define REDISMODULE_CONFIGCHANGE_VERSION 1
+typedef struct RedisModuleConfigChange {
+    uint64_t version;       /* Not used since this structure is never passed
+                               from the module to the core right now. Here
+                               for future compatibility. */
+    uint32_t num_changes;   /* how many redis config options where changed */
+    const char **config_names; /* the config names that were changed */
+} RedisModuleConfigChangeV1;
+
 #define RedisModuleModuleChange RedisModuleModuleChangeV1
```

Comment:
nit pick, maybe move this up to be next to RedisModuleModuleChangeV1 and create a similar alias for the new one.

Reply:
yes, that was a mistake on my part.  meant to do that.  think what I have now does that.